src/ontology/uberon-edit.obo

```diff
@@ -224439,6 +224439,18 @@ relationship: dc-contributor https://orcid.org/0000-0002-9185-3994
 relationship: part_of UBERON:0000101 ! lobe of lung
 property_value: dcterms-date "2023-07-20T12:43:29Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600018
+name: neuroendocrine system
+def: "A regulatory system of the body that integrates the nervous system and the endocrine system. This system involves specialized neuroendocrine cells primarily located in the hypothalamus, pituitary gland, and various organs. The neuroendocrine system functions through the release of neurotransmitters and neurohormones, enabling communication between these systems to govern vitally important processes that include growth, reproduction, metabolism and energy homeostasis, electrolyte and water balance, and responses to stress." [doi:10.1016/B978-0-12-375097-6.10001-0, PMID:15481802, Wikipedia:endocrinology]
```

Comment:
Beware that this definition (specifically the part “involves […] cells **primarily located in the hypothalamus, pituitary gland**, and various organs”) makes the term vertebrate-specific. Invertebrates may lack a hypothalamus or a pituitary gland but they do have a “neuroendocrine system” (see for example [Hartenstein 2006](https://doi.org/10.1677/joe.1.06964)), so if it is intended that this term should be vertebrate-specific then this should be clearly stated; otherwise the definition should avoid referring to vertebrate-specific structures or at least use precautions around them.

Reply:
Thank you @gouttegd, the intention is that is not vertebrate-specific, and anything that is vertebrate-specific will be in a comment.